buildtools_test.go

```diff
@@ -353,6 +353,103 @@ func TestGoBuildInfo(t *testing.T) {
 	cleanGoTest(gopath)
 }
 
+func TestGoBuildnameAndNumberFromEnv(t *testing.T) {
```

Comment:
TestGoBuildnameAndNumberFromEnv --> TestGoBuildNameAndNumberFromEnv

Can we reuse the code in the *TestGoBuildInfo* test? I think we should avoid this code duplication.

Reply:
I deleted the test because the native commands flow is already tested in [unit test](https://github.com/jfrog/jfrog-cli/pull/453/files#diff-3f73e7bf71fef561eb5790dc08c1c68bR161).